src/main/java/com/compilercharisma/chameleonbusinessstudio/controller/AppointmentController.java

```diff
@@ -55,22 +55,25 @@ public AppointmentController(
      * 
      * @return 
      */
-//    // https://stackoverflow.com/a/63966321
-//    @GetMapping(path="available")
-//    public ResponseEntity<PagedModel<EntityModel<AppointmentEntity>>> getAvailableInDays(
-//            @RequestParam(required=false, defaultValue="30") int days,
-//            Pageable page
-//    ){
-//        LocalDateTime now = LocalDateTime.now();
-//        LocalDateTime later = now.plusDays(days);
-//
-//        Page<AppointmentEntity> entities = appointments.getAvailableAppointments(now, later, page);
-//
-//        return ResponseEntity
-//                .ok()
-//                .contentType(MediaTypes.HAL_JSON)
-//                .body(asm.toModel(entities, modelAssembler));
-//    }
+    // https://stackoverflow.com/a/63966321
+    @GetMapping(path="available")
+    public Mono<ResponseEntity<PagedModel<EntityModel<AppointmentEntity>>>> getAvailableInDays(
+            @RequestParam(required=false, defaultValue="30") int days,
+            Pageable page
+    ){
+        throw new UnsupportedOperationException("Need to reimplement paging");
```

Comment:
I'm assuming the pagination algorithm is changing due to Web Flux upgrade?

Reply:
This isn't Webflux's fault this time, it's Vendia. The implementations Spring Boot generated for our repositories implemented paging handling, but we've yet to implement that into our custom Vendia repository